docs/users/extend/customization-extendibility.md

```diff
@@ -22,7 +22,21 @@ The default web container for ddev uses NGINX as the web server. A default confi
 - **NOTE:** The "root" statement in the server block must be `root $NGINX_DOCROOT;` in order to ensure the path for NGINX to serve the project from is correct.
 - Save your configuration file and run `dev start` to start the project environment. If you encounter issues with your configuration or the project fails to start, use `ddev logs` to inspect the logs for possible NGINX configuration errors (or use `ddev ssh` and inspect /var/log/nginx/error.log.)
 - Any errors in your configuration may cause the web container to fail and try to restart, so if you see that behavior, check your container.
-- Changes to .ddev/nginx-site.conf take place only after you do a `ddev rm` followed by `ddev start`.
+- **IMPORTANT**: Changes to .ddev/nginx-site.conf take place only after you do a `ddev rm` followed by `ddev start`.
+
+## Providing custom PHP configuration (php.ini)
+
+You can provide additional PHP configuration for a project by creating a directory called `.ddev/php/` and adding any number of php configuration ini files. Normally, you should just override the specific option that you need to override. Note that any file that exists in `.ddev/php/` will be copied into `/etc/php/[version]/(cli|fpm)/conf.d`, so it's possible to replace files that already exist inside the container, so if in doubt, put your custom overrides in a file called `my-php.ini`.
+
+One interesting implication of this behavior is that it's possible to disable extensions by replacing the configuration file that loads them. For instance, if you were to create an empty file at `.ddev/php/20-xdebug.ini`, it would replace the configuration that loads xdebug, which would cause xdebug to not be loaded!
+
+To load the new configuration, just run a `ddev stop` and `ddev start`.
+
+## Providing custom mysql/MariaDB configuration (my.cnf)
+
+You can provide additional PHP configuration for a project by creating a directory called `.ddev/mysql/` and adding any number of MySQL configuration files. These files will be automatically included when MySQL is started.
+
+To load the new configuration, just run a `ddev stop` and `ddev start`.
```

Comment:
`ddev restart` right? As would be expected, `ddev restart` does work, just tested it.